content/docs/accessibility.md

```diff
@@ -91,7 +92,7 @@ function Glossary(props) {
 }
 ```
 
-When you don't need any props on the Fragment tag you can use the [short syntax](/docs/fragments.html#short-syntax), if your tooling supports it:
+Երբ դուք կարիք չունեք որևէ prop լինի Fragment tag-ի վրա, դուք կարող եք օգտագործել [կարճ գրառումը](/docs/fragments.html#short-syntax), եթե ձեր գործիքները նպաստում են դրան\`
```

Comment:
երկու բան.

«դուք»-երը աշխատի քիչ օգտագործես, հիմնականում չի թարգմանվում «you»-ն։
հիմանականում միջակետ ենք դնում : <- սրա փոխարեն, ոչ թե բութ

Երբ ԴՈՒՔ կարիք չունեք որևէ......, ԴՈՒՔ կարող եք....

երկու դեպքում էլ «դուք»-ը պետք չի. «ունԵՔ»-ը ու «կարող ԵՔ»-ը լրացնում են «դուք»-ին։

Երբ կարիք չունեք, որպեսզի որևէ prop լինի Fragment թեգի վրա, կարող եք օգտագործել [կարճ գրառումը](/docs/fragments.html#short-syntax), եթե ձեր գործիքները նպաստում են դրան.